v3/plugins/ms-vscode/vim/v1.12.0/meta.yaml

```diff
@@ -0,0 +1,15 @@
+publisher: VSCodeVim
```

Comment:
publisher id and folder should be the same

else you've an error on build like

!!! Location mismatch in plugin 'VSCodeVim/vim/latest':
!!!   Expected location: 'v3/plugins/VSCodeVim/vim/latest/meta.yaml'
!!!   Actual location: 'v3/plugins/ms-vscode/vim/latest/meta.yaml'